packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -503,6 +503,40 @@ class VmServiceWrapper implements VmService {
     }
   }
 
+  /// The `getHttpProfile` RPC is used to retrieve HTTP profiling information
+  /// for requests made via `dart:io`'s `HttpClient`.
+  ///
+  /// The returned [HttpProfile] will only include requests issued after
+  /// [httpTimelineLogging] has been enabled or after the last
+  /// [clearHttpProfile] invocation.
+  Future<HttpProfile> getHttpProfile(String isolateId, {int updatedSince}) {
+    return trackFuture(
```

Comment:
don't we need to add `isProtocolVersionSupported` checks around all of these to make sure we don't try to call this method on an old version of the vm_service?

Reply:
Good call, yes we do.